symengine/cwrapper.cpp

```diff
@@ -737,14 +737,22 @@ void vecbasic_free(CVecBasic *self)
     delete self;
 }
 
-void vecbasic_push_back(CVecBasic *self, const basic value)
+CWRAPPER_OUTPUT_TYPE vecbasic_push_back(CVecBasic *self, const basic value)
 {
+    CWRAPPER_BEGIN
+
     self->m.push_back(value->m);
+
+    CWRAPPER_END
 }
 
-void vecbasic_get(CVecBasic *self, int n, basic result)
+CWRAPPER_OUTPUT_TYPE vecbasic_get(CVecBasic *self, int n, basic result)
 {
+    CWRAPPER_BEGIN
+
     result->m = self->m[n];
+
```

Comment:
@abinashmeher999 perhaps this is an unpopular opinion, but I would much rather it be `int` than `unsigned int`.

If it is `int`, then negative numbers in indices will get caught properly, unlike wrapping around and becoming some positive number that (maybe) valid.